bootstrap.d/meta-pkgs.yml

```diff
@@ -2,7 +2,9 @@ packages:
   # Meta package with the bare minimum to boot kmscon and be useful
   - name: base
     labels: [aarch64]
-    from_source: managarm
+    source:
+      subdir: ports
```

Comment:
I wonder if we should put these into `ports` or add a `meta-sources` subdirectory (xbstrap should create such a directory automatically, right?).

Reply:
As far as I know yes it should. Moving it to `meta-sources` sounds good to me, I’ll implement that change tomorrow.